Assets/Scripts/TextScroll.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class TextScroll : MonoBehaviour
+{
+    public float scrollSpeed = 50f;
+    private RectTransform rectTransform;
+    private TextMeshProUGUI textComponent;
+
+    /*private void Update()
```

Comment:
Is this still needed?

Reply:
Talked about in VC, waiting for commit to remove it

Reply:
Removed the script as its useless at the moment, will possibly add in UX PR.